package.json

```diff
@@ -18,7 +18,8 @@
     "parse-kfc": "node parser_kfc.js",
     "parse-mcd": "node parser_mcd.js",
     "parse-nfp": "node parser_nfp.js",
-    "setup": "npm run parse-kfc && npm run parse-mcd && npm run parse-nfp && npx knex seed:run && npm run dev",
+    "db:create": "echo Creating new DB && node create_db.js",
```

Comment:
LOL... might wanna change the `echo` to `console.log` in `create_db.js`.

Reply:
> LOL... might wanna change the `echo` to `console.log` in `create_db.js`.

I was thinking that it's random too. Okay will do 😂